.travis.yml

```
@@ -13,7 +13,7 @@ matrix:
 
 env:
   - TEST_SUITE=static
-  - TEST_SUITE=unit
+  - TEST_SUITE=unit 
```

```suggestion
  - TEST_SUITE=unit
```

unnecessary whitespace change